### ISSUE TITLE
import with open_mfdataset: sort filenames

### ISSUE THREAD

**Issue by ghislainp** [intake/intake-xarray#64]
I have one nc file per year.. and many years. I use glob wildcard to trigger open_mfdataset. The problem is that when filenames are not sorted (here by year), the time dimension in the xarray is not sorted. I used to sort the files when directly calling open_mfdatasets, but here how to do it in the yml file ?
Can we add a sort_files option to intake_xarray.netcdf.NetCDFSource ?

**Comment by martindurant** [comment]
Would you mind checking if this is still the case on master?

**Comment by ghislainp** [comment]
I will later today, but I checked the master (netcdf.py) and I see the combined='nested' default which presumably solves this problem. This raises a related problem that I faced yesterday. I don't open another issue because I think both a related.
I spent a lot of time figuring out why my code with open_mfdataset was working while the intake version was not. Then looking at the code, I saw: The default concat_dim='concat_dim'. Similar to combined='nested'.

The defaults for open_mfdataset are diffirent and in my case are in fact better, as I can open my multi-file dataset without any options. To make this work with intake I had to add in my catalog "concat_dim: _not_supplied" which is a bit ugly. I fair that this _not_supplied value may change in the future. The combined='nested' default in  the master of intake_xarray is a similar issue, I'll have to add combined='coords' to obtain the default behavior of open_mfdataset.

So my question: why not to keep the default behavior of open_mfdataset which is well documented ? If not, this would need to be documented in xarray_intake.
